checker/src/main/java/org/checkerframework/checker/calledmethods/CalledMethodsChecker.java

```diff
@@ -0,0 +1,94 @@
+package org.checkerframework.checker.calledmethods;
+
+import java.util.LinkedHashSet;
+import org.checkerframework.common.basetype.BaseTypeChecker;
+import org.checkerframework.common.returnsreceiver.ReturnsReceiverChecker;
+import org.checkerframework.common.value.ValueChecker;
+import org.checkerframework.framework.qual.StubFiles;
+import org.checkerframework.framework.source.SupportedOptions;
+import org.checkerframework.framework.source.SuppressWarningsPrefix;
+
+/**
+ * The Called Methods Checker tracks the methods that have definitely been called on an object. One
+ * common use case for the Called Methods Checker is to specify safe combinations of options to
+ * builder or builder-like interfaces, preventing objects from being instantiated incompletely.
+ */
+@SuppressWarningsPrefix({
+    // Preferred checkername.
+    "calledmethods",
+    // Deprecated checkernames, supported for backward compatibility.
+    "builder",
+    "object.construction",
+    "objectconstruction"
+})
+@SupportedOptions({
+    CalledMethodsChecker.USE_VALUE_CHECKER,
+    CalledMethodsChecker.COUNT_FRAMEWORK_BUILD_CALLS,
+    CalledMethodsChecker.DISABLE_BUILDER_FRAMEWORK_SUPPORTS,
+})
+@StubFiles({"DescribeImages.astub", "GenerateDataKey.astub"})
+public class CalledMethodsChecker extends BaseTypeChecker {
+
+    /**
+     * If this option is supplied, count the number of analyzed calls to build() in supported
+     * builder frameworks and print it when analysis is complete. Useful for collecting metrics.
+     */
+    public static final String COUNT_FRAMEWORK_BUILD_CALLS = "countFrameworkBuildCalls";
+
+    /**
+     * This option can be used to disable the support (and therefore the automated checking of) code
+     * that uses the given builder frameworks. Useful when a user **only** wants to enforce
+     * specifications on custom builder objects (such as the AWS SDK examples).
+     */
+    public static final String DISABLE_BUILDER_FRAMEWORK_SUPPORTS =
+            "disableBuilderFrameworkSupports";
+
+    /**
+     * The key for the {@link #DISABLE_BUILDER_FRAMEWORK_SUPPORTS} option to disable Lombok support.
+     * Use it via {@code -AdisableBuilderFrameworkSupports=LOMBOK}.
+     */
+    public static final String LOMBOK_SUPPORT = "LOMBOK";
```

Comment:
This string only needs to be used once or maybe in the code, and it is unlikely to change.  (Also, the name `LOMBOK_SUPPORT` is a bit misleading since this is a string or name, not a BuilderFrameworkSupport.)  Just inline it where it is used.  That will simplify the code and make it easier to read.